setup.py

```diff
@@ -29,13 +29,14 @@
 
 from setuptools import setup, find_packages
 
-tests_require = ['asynctest', 'nose']
+tests_require = ['async-solipsism', 'pytest', 'pytest-asyncio', 'pytest-mock']
```

Comment:
Yes, I thought you might like it for kattelmod. Although I think what I put in place for kattelmod is in some ways more flexible because it can handle a clock that runs at a fixed ratio to real-time while still interacting with the outside world.